retrospective-forecasting/main.py

```diff
@@ -181,49 +167,72 @@
     eval_dir = ValidPath(config["evaluation"]["save_dir"])
 
     scores = []
-    for metric_name in config["evaluation"]["metrics"]:
-        metric_function = linmod.eval.__dict__[metric_name]
 
-        for forecast_path in forecast_dir.glob("forecasts_*.parquet"):
-            model_name = forecast_path.stem.split("_")[1]
-            print_message(
-                f"Evaluating {model_name} model using {metric_name}...", end=""
+    for forecast_path in forecast_dir.glob("forecasts_*.parquet"):
```

Comment:
this is something that would be better implemented in a makefile (or snakefile)

although if it's fast enough, then whatever

out of scope for this PR anyway